wiki-jpa-dao/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml

```diff
@@ -37,6 +37,10 @@
   </changeSet>
 
   <changeSet author="wiki" id="1.0.0-2">
+    <validCheckSum>7:7209dfb03ea324f8cea261daa5b24b78</validCheckSum>
+    <validCheckSum>7:131376aa9a40e20b14bf227f37c8d309</validCheckSum>
+    <validCheckSum>7:c046fcb3d84bc510622a067bc1a949d8</validCheckSum>
+    <validCheckSum>7:7b342dc7f44b14421cabf99ac96dafa4</validCheckSum>
```

Comment:
Why do we need all these checksums ? Only the checksum of the new changeset should be sufficient, no ?